website/src/lib/worker.ts

```diff
@@ -0,0 +1,100 @@
+import { compileMemfs } from 'browser-bundler';
```

Comment:
will publish this soon, currently, a local link, based on https://github.com/hardfist/neo-tools with some bug fixes